### ISSUE TITLE
How to download

### ISSUE THREAD

**Issue by SophisticatedFox4** [ndless-nspire/Ndless#353]
After I have downloaded Ndless, should I put all of the gba files into .gba.tns format?

**Comment by Vogtinator** [comment]
Yes, you need to rename `.gba` files into `.gba.tns` to be able to transfer them to the calculator.

**Comment by SophisticatedFox4** [comment]
Oh, thank you! Perhaps you should add that to Readme.md?
